feature/home/presentation/home-ui/src/main/java/com/chocolatecake/ui/people/adapters/TvShowsByPeopleAdapter.kt

```diff
@@ -0,0 +1,11 @@
+package com.chocolatecake.ui.people.adapters
+
+import com.chocolatecake.bases.BaseAdapter
+import com.chocolatecake.ui.home.R
+import com.chocolatecake.viewmodel.people.PeopleDetailsListener
+import com.chocolatecake.viewmodel.people.PeopleDetailsUiState
+
+
+class TvShowsByPeopleAdapter{
+
+}
```

Comment:
where is the implementation and why it does not inherit from the base adapter ??